### PR TITLE
refactor(task): add org id to log reader calls

### PR DESCRIPTION
**Pull request by lyondhill**
To write a query log reader we need to have the organization.

### CODE REVIEW

File: task.go [hunk 1]
```diff
@@ -85,6 +86,7 @@ type RunFilter struct {
 
 // LogFilter represents a set of filters that restrict the returned results
 type LogFilter struct {
+	Org  *ID
```
[the remark this answers is not in the document]
Comment (lyondhill) [reply task.go]: Im fine with putting it somewhere else. It just needs to be somewhere so i can query the database.

[the remark this answers is not in the document]
Comment (goller) [reply task.go]: tokens will be applied to a single org... I think the single org token should make it way easier for us to use.

[the remark this answers is not in the document]
Comment (mark-rushakoff) [reply task.go]: Okay, we'll get the org from the token once that's ready.

[the remark this answers is not in the document]
Comment (lyondhill) [reply task.go]: I will put in some TODO's as a reminder that this is temporary.

**Pull request merged**